SU-HSDO/suhumsci
# SHS-5116: Move editoria11y button

# READY FOR REVIEW

## Summary
Change position of editoria11y component

## Need Review By (Date)
11/9

## Urgency
medium

## Steps to Test
1. Log in as a "Developer" (admin) user
2. Confirm that the editoria11y component is rendered on top of the "Back to Top" button on the right bottom corner of the page and that both are clickable at the same time

<img width="183" alt="Screenshot 2023-11-09 at 08 32 58" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/beb2917c-2167-4ed9-8d62-3075cc3ae1b8">


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
